src/Tabs/index.js

```diff
@@ -56,10 +62,16 @@ Tabs.propTypes = {
    */
   children: PropTypes.node.isRequired,
   /**
-   * Sets default tab selection by index in source order
+   * Sets _default_ tab selection by index in source order
    */
   defaultSelectedIndex: PropTypes.number,
-  /** Callback invoked with `tabId` of the tab as the argument */
+  /**
+   * Sets selected tab by index, making Tabs **fully controlled**.
+   * When using this prop, you must use the `onTabChange` callback
+   * to update the value of this prop to update the selected tab.
+   */
+  selectedIndex: PropTypes.number,
+  /** Callback invoked with the index of the tab the user is moving selection to */
```

Comment:
The `onTabChange` callback has a different argument now... this might be considered a breaking change, but given the fact that Tabs were uncontrolled previously, this callback was functionally useless until now